jobs/test-paired-bam/job.json

```diff
@@ -1,18 +1,16 @@
 {
     "cwl_wf_url": "https://github.com/Sage-Bionetworks-Workflows/dockstore-workflow-rnaseq",
-    "cwl_args_url": "https://raw.githubusercontent.com/Sage-Bionetworks-Workflows/dockstore-workflow-rnaseq/master/jobs/test-paired-fastq/job.json",
-    "index_synapseid": "syn20835938",
-    "nthreads": 15,
-    "synapse_parentid": "syn20783166",
+    "cwl_args_url": "https://raw.githubusercontent.com/Sage-Bionetworks-Workflows/dockstore-workflow-rnaseq/tests/jobs/test-paired-bam/job.json",
```

Comment:
Do we have a "chicken or the egg" problem here? We can anticipate what the new URL will be for the master branch if this were to get merged now.

Reply:
I'd go ahead and change it to the master branch url